src/strings/en-US/PowerFxResources.resw

```diff
@@ -2265,6 +2265,53 @@
   <data name="AboutChar_column_of_numbers" xml:space="preserve">
     <value>A column of code numbers from the character set on your platform.</value>
   </data>
+  <data name="AboutParseJson" xml:space="preserve">
+    <value>Converts a JSON string into a loosely-typed object.</value>
```

Comment:
"loosely typed" may not be the language we want to use publicly for this

Reply:
Changed the language